src/components/FileList/FacetsRenderer.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+/**
+ * Specific renderer for Facets
+ * 
+ */
+export default class FacetsRenderer extends Component {
+  render() {
+    const { item } = this.props;
+    return (
+      <div>
+        {item.value}
+      </div>
+    )
+  }
+}
```

Comment:
@jeffmcaffer thoughts on using the functional style of components when they are stateless? It's more concise. I'd write this as 
```js
const FacetsRenderer = ({item}) => <div>{item.value}</div>;
```
Internally we've been using `flow` for prop validation, but with prop types I think you'd do this
```js
FacetsRenderer.propTypes = { item: PropTypes.object };
export default FacetsRenderer;
```

Reply:
I do like the conciseness of the functional style components. OTOH I like the regularity of having a class. I'm flexible here but would like there to be some logic to the choice. Eg. if small and stateless, functional, else class. (whatever)